packages/elements/src/pagination/index.ts

```diff
@@ -442,6 +423,42 @@ export class Pagination extends BasicElement {
    */
   private onInputFocusedChanged (event: FocusedChangedEvent): void {
     this.inputFocused = event.detail.value;
+
+    if (!this.inputFocused) {
+      this.updatePageInput();
+    }
+  }
+
+  /**
+   * Update page by using value from the input
+   * @returns {void}
+   */
+  private updatePageInput (): void {
+    // Prevent update page to the same value
+    if (this.value === this.input.value) {
+      return;
+    }
+
+    const oldValue = this.value;
+    let newValue = parseInt(this.input.value, 10);
+
+    // Reset input and boundary value into supported range.
+    if (this.validatePage(this.input.value)) {
+      if (newValue <= 0) {
+        newValue = 1;
+      }
+      else if (newValue > this.internalMax) {
+        newValue = this.internalMax;
+      }
+      this.value = newValue.toString();
+    }
+    else if (!isNaN(newValue)) {
+      this.value = '1';
+    }
```

Comment:
Why do we need this? We can just leave it as the old value.

Reply:
We need this and it should reset to be '1' this is correct because the behavior of the function `updatePageInput` which wrap this condition is used for user interaction, so when a user input number is less than 1 it should reset to '1' and vice versa if the user input number more than max it should reset to max value. 😀

Reply:
It shouldn't reset though - it should revert to the previous valid value.

| Current Value | Input | Value |
|---|---|---|
| `''` | "8.t" | `''` |
| `'2'` | "Typo" | `'2'` |
| `'2'` | "30" | `'30'` |

cc: @dtanp-rft

Reply:
@Theeraphat-Sorasetsakul   Can you please take care of this one? just a minor issue. It needs to be reverted to the empty string.

Reply:
I already create a PR https://github.com/Refinitiv/refinitiv-ui/pull/142 to fix this issue. Could you please continue at that PR.